### ISSUE TITLE
Migrating Pull Requests from Bitbucket to Github

### ISSUE THREAD

**Issue by jsantos** [viperproject/bitbucket-issue-migration#47]
Hi, first of all thank you for this amazing tool! Feel free to close this right away because it's not really an issue, but simply a question.

I'm test-driving to move one of my organisation's repositories to Github, but in the end all existing pull requests (already merged) are converted into issues. Is it meant to be like this? 

I've been digging into the code and noticed that if a pull request is closed it automatically creates an issue (and closes it) on its behalf. I understand that actually creating a pull request that has already been merged is not trivial at all, just wanted to know if this is the default behaviour or if I'm doing something wrong :)

Thanks a lot!

**Comment by fpoli** [comment]
> I'm test-driving to move one of my organisation's repositories to Github, but in the end all existing pull requests (already merged) are converted into issues. Is it meant to be like this?

Yes, it's the intended behavior because creating closed pull requests is more difficult.

**Comment by jsantos** [comment]
@fpoli Got it! Thanks for your confirmation!

I'm also bumping into an API error when I have "assignees" mapped to certain PR's that are not on the repository anymore (e.g. ex-employees):

`github.GithubException.GithubException: 422 {"message": "Validation Failed", "errors": [{"value": "<contributor_username>", "resource": "Issue", "field": "assignees", "code": "invalid"}], "documentation_url": "https://docs.github.com/rest/reference/issues#create-an-issue"}`

This only happens when I have the mappings for `USER_MAPPING` on `config.py`. Maybe it's a Github API limitation, just wondering if you bumped into something similar before.

**Comment by danedf** [comment]
> I'm also bumping into an API error when I have "assignees" mapped to certain PR's that are not on the repository anymore (e.g. ex-employees):
> 
> `github.GithubException.GithubException: 422 {"message": "Validation Failed", "errors": [{"value": "<contributor_username>", "resource": "Issue", "field": "assignees", "code": "invalid"}], "documentation_url": "https://docs.github.com/rest/reference/issues#create-an-issue"}`
> 
> This only happens when I have the mappings for `USER_MAPPING` on `config.py`. Maybe it's a Github API limitation, just wondering if you bumped into something similar before.

This may in, in fact, be a GitHub API issue. I've been unable to fix this issue using:
* Python/requests
* PyGithub

I've only been able to proceed by setting `assignees = []`. If I put any values in there, even in a properly formatted list, the GitHub API seems to manipulate it and treat it like a string regardless. I also have not had luck using `assignee = 'USERNAME'` with a single user, unfortunately.
